x-pack/test/scalability/apis/api.telemetry.cluster_stats.no_cache.1600_dataviews.json

```diff
@@ -0,0 +1,47 @@
+{
+  "journeyName": "POST /api/telemetry/v2/clusters/_stats - no cache - 1600 dataviews",
+  "scalabilitySetup": {
```

Comment:
<img width="1129" alt="image" src="https://user-images.githubusercontent.com/10977896/220422444-0654c909-91a2-4b81-bfbe-32cc98feba2e.png">

Reply:
++ I'm concerned about the `Connection refused` failures, though... shouldn't those be timeouts instead?

Reply:
Timeouts increased in [elastic/kibana@`98c1f84` (#151110)](https://github.com/elastic/kibana/pull/151110/commits/98c1f841acfaa022600991d19c666b734457fa63)

Reply:
@afharo I think this test basically kills Kibana server and that's why we see "Connection refused"

I can see multiple errors like 
```
[2023-02-21T18:51:35.148+01:00][ERROR][plugins.dataViews.dataView.indexPatterns] ConnectionError: connect EADDRNOTAVAIL 127.0.0.1:9220 - Local (0.0.0.0:0)
    at KibanaTransport.request (/Users/dmle/github/kibana/node_modules/@elastic/transport/src/Transport.ts:585:17)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at KibanaTransport.request (create_transport.ts:57:17)
    at ClientTraced.FieldCapsApi [as fieldCaps] 

(/Users/dmle/github/kibana/node_modules/@elastic/elasticsearch/src/api/api/field_caps.ts:78:10)
    at callFieldCapsApi (es_api.ts:74:12)
    at getFieldCapabilities (field_capabilities.ts:46:23)
    at IndexPatternsFetcher.getFieldsForWildcard (index_patterns_fetcher.ts:76:31)
    at IndexPatternsApiServer.getFieldsForWildcard (index_patterns_api_client.ts:33:12)
    at DataViewsService.getFieldsAndIndicesForWildcard (data_views.ts:541:12)
    at DataViewsService.refreshFieldSpecMap (data_views.ts:624:46)
    at DataViewsService.initFromSavedObjectLoadFields (data_views.ts:756:33)
    at DataViewsService.initFromSavedObject (data_views.ts:798:34)
```

and 

```
2023-02-16T16:01:15.143+01:00][ERROR][http] NoLivingConnectionsError: There are no living connections
    at KibanaTransport.request (/Users/dmle/github/kibana/node_modules/@elastic/transport/src/Transport.ts:456:17)
    at KibanaTransport.request (/Users/dmle/github/kibana/node_modules/elastic-apm-node/lib/instrumentation/modules/@elastic/elasticsearch.js:143:28)
    at KibanaTransport.request (create_transport.ts:57:29)
    at Security.hasPrivileges (/Users/dmle/github/kibana/node_modules/@elastic/elasticsearch/src/api/api/security.ts:962:33)
    at checkPrivilegesAtResources (check_privileges.ts:121:81)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at Object.atSpaces (check_privileges.ts:219:16)
    at SavedObjectsSecurityExtension.checkSavedObjectsPrivileges [as checkPrivilegesFunc] (check_saved_objects_privileges.ts:52:20)
    at SavedObjectsSecurityExtension.checkPrivileges (saved_objects_security_extension.ts:612:14)
    at SavedObjectsSecurityExtension.checkAuthorization (saved_objects_security_extension.ts:411:45)
    at SavedObjectsSecurityExtension.authorize (saved_objects_security_extension.ts:578:54)
    at SavedObjectsSecurityExtension.authorizeGet (saved_objects_security_extension.ts:822:12)
    at SavedObjectsRepository.get (repository.ts:1704:33)
    at SavedObjectsClient.get (saved_objects_client.ts:117:12)
    at UiSettingsClient.read (ui_settings_client_common.ts:158:20)
    at UiSettingsClient.getUserProvided (ui_settings_client_common.ts:53:59)
    at UiSettingsClient.getAll (base_ui_settings_client.ts:56:26)
    at Object.fieldFormatServiceFactory (plugin.ts:44:31)
    at getVisData (get_vis_data.ts:35:30)
    at vis.ts:36:23
    at Router.handle (router.ts:192:30)
    at handler (router.ts:147:13)
    at exports.Manager.execute (/Users/dmle/github/kibana/node_modules/@hapi/hapi/lib/toolkit.js:60:28)
    at Object.internals.handler (/Users/dmle/github/kibana/node_modules/@hapi/hapi/lib/handler.js:46:20)
    at exports.execute (/Users/dmle/github/kibana/node_modules/@hapi/hapi/lib/handler.js:31:20)
    at Request._lifecycle (/Users/dmle/github/kibana/node_modules/@hapi/hapi/lib/request.js:371:32)
    at Request._execute (/Users/dmle/github/kibana/node_modules/@hapi/hapi/lib/request.js:281:9)
```

Reply:
We covered this on Zoom. The intention of this test is to show that this API is problematic in this scenario. It will help us track any improvements for different efforts that have branched off since we've identified this bottleneck.